sagemaker-huggingface/text-summarizer/huggingface-custom-text-summarizer.ipynb

```diff
@@ -0,0 +1,2626 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        "Alex: Were you able to attend Friday night's basketball game?\nBenjamin: I was unable to make it.\nAlex: You should have been there. It was intense.\nBenjamin: Is that right. Who ended up winning?\nAlex: Our team was victorious.\nBenjamin: I wish I was free that night. I'm kind of mad that I didn't go.\nAlex: It was a great game. Everything alright tough?\nBenjamin: Yeah man thanks for asking, it's just that my mom is sick and I am taking care of her.\nAlex: Oh sorry to hear that. Hope she makes a fast recovery :muscle:\nBenjamin: She will, she just has a nasty flu but she will be alright :D\nAlex: Glad to hear that!\nBenjamin: What was the score at the end of the game?\nAlex: Our team won 101-98.\nBenjamin: Sounds like it was a close game then.\nAlex: That's the reason it was such a great game.\nBenjamin: I'll go to the next one for sure.\nAlex: It's next weekend so you better put on your calendar ahaha\nBenjamin: ahaha I will I will. Talk to you later!\nAlex: Alright! Tell your mom I hope she gets better quickly."
</pre><p>Is this a typo: <code>Everything alright tough?</code>  ? Should it be <code>Everything alright though</code>?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>